src/lib/content.d.ts

```diff
@@ -544,3 +553,32 @@ interface CampaignFieldSelect extends CampaignField {
 		value: string;
 	}[];
 }
+
```

Comment:
Shall we add a label defining where these types are used? Right not they've been put under the 'Callout section' but I think we should create their own one.